ccd-definition/CaseEvent.json

```diff
@@ -389,12 +389,26 @@
     "ID": "NOTIFY_APPLICANT_SOLICITOR1_FOR_FAILED_PAYMENT",
     "Name": "Notify failed payment",
     "Description": "Notify applicant solicitor 1 for failed payment",
-    "DisplayOrder": 26,
+    "DisplayOrder": 27,
     "PreConditionState(s)": "*",
     "PostConditionState": "*",
     "SecurityClassification": "Public",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-submit",
     "ShowSummary": "N",
     "ShowEventNotes": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "GENERATE_CLAIM_FORM",
+    "Name": "Generate claim form",
+    "Description": "Generates claim form after successful payment",
```

Comment:
I have run past our suggestion with Emily already - she agreed with our approach. The event description we want to see in the history is going to be something along the lines of `Claim issued` (which comes after `Claim created`).  `CASE_ISSUED` is also going to be renamed to something like `AWAITING_CONFIRM_SERVICE`.

All of which will be part of another story.

Reply:
just for our knowledge, name is added on the event history, description I belive is what would be read out on a screen reader (and when you hover over the event in next step dropdown)